modules/ROOT/pages/ip-allowlist.adoc

```diff
@@ -0,0 +1,72 @@
+= IP Allowlist
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: IP, allowlist, validation, policy
+
+[width="100%", cols="5,15"]
+|===
+>s| Policy Name | IP Allowlist
+>s|Summary      | Allows a list or range of specified IP addresses to request access   
+>s|Category | Security
+>s|First Mule version available | v3.8.0
+.1+>.^s| Returned Status Codes
+| 403 - IP is rejected
+|===
+
+
+The IP Allowlist policy allows a list or a range of specified IP addresses to access a configured API endpoint.
+
+The IP Allowlist policy allows access to a protected resource when a match is found between a source IP (specified when configuring the policy) and a list of individual IPs or range of IPs.
+
+Based on your business need, you can specify a DataWeave 2.0 expression for Mule runtime engine 4 (Mule) or MEL expression for Mule 3. You specify the expression to define the source IP to be used when the policy tries to determine whether the IP belongs to the restricted IP list defined in the policy.
+
+
+[[specifying_ip_address]]
+== Specifying IP Addresses to Allowlist in Your Policy 
+
+You can specify the IP addresses to approve in the policy in the following ways:
+
+* Allowlist access based on the IP address of the request.
+* Allowlist access based on the origin IP address that is determined by resolving a specific DataWeave or MEL expression, such as the https://en.wikipedia.org/wiki/X-Forwarded-For[x-Forwarded-For] header.
+* Any other origin.
+
+=== Allowlisting Access Based on the IP address of the Request
+
+To allowlist access based on the IP address of the request, you can:
+
+* Define a specific IP address by enumerating it in the Allowlist field, for example, `192.168.1.1`.
+* Define a subset of addresses by identifying a subnet mask. 
+For example, `192.168.3.1/30`, which includes the consolidated range `192.168.3.01` through `192.168.3.3`.
+* Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. 
++
+For example, setting `192.168` includes IP addresses from `192.168.0.0` through `192.168.255.255`.
+
+=== Allowlisting Access Based on the Origin IP Address of the x-Forwarded-For Header
+
+If the client connects to your API through an HTTP proxy or a load balancer, you can allowlist the client's specific IP address (the IP address originating the request) instead of the address that appears in the request.
```

Comment:
Why would the audience want or need to use the specific address vs. the origin address?